docs/src/windows.md

```diff
@@ -0,0 +1,106 @@
+# Install and contribute to Erdapfel on Windows
+
+## Windows 7 / 8
+
+- Install React Dev Tools in your browser ([chrome](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi) / [Firefox](https://addons.mozilla.org/fr/firefox/addon/react-devtools/))
+- Install [VirtualBox](https://www.virtualbox.org/)
+- Mount [this xubuntu image](http://cdimages.ubuntu.com/xubuntu/releases/18.04/release/xubuntu-18.04.1-desktop-amd64.iso)
+- Allocate at least 2 CPU, 4GB of RAM and 20GB of HDD and material acceleration to the VM
+- In settings > Network, choose Bridge Adapter
+- Inside the VM, install an IDE (enable ESlint plugin), open a terminal
+- Clone erdapfel in the folder of your choice (ex: "/home/{username}/www/"): 
+```
+sudo apt install git
+git clone https://github.com/QwantResearch/erdapfel.git
+```
+- Enter the [commands](#commands) below to build and run Erdapfel
+- Retrieve your IP with ```ifconfig``` (just after "inet ...")
+<br>Your local instance of Erdapfel will be visible at **http://localhost:3000** in the VM or **http://{ip}:3000** outside of the VM
+
+# Windows 10
+
+- Install React Dev Tools in your browser ([chrome](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi) / [Firefox](https://addons.mozilla.org/fr/firefox/addon/react-devtools/))
+- Install [NodeJS LTS](https://nodejs.org/en/download/)
+- Install [Git](https://git-scm.com/downloads)
+- Install [Python](https://www.python.org/downloads/)
+- Install an IDE (enable ESlint plugin)
+- From an Admin account, enable developer mode (Start menu > Settings > Updates & Security > For developers > check "Developer mode")
+<br>And enable WSL (Settings > Apps > Programs and features > Turn Windows features on or off > enable "Windows Subsystem for Linux")
+- Then from any account, install [Ubuntu from Microsoft Store](https://www.microsoft.com/fr-fr/p/ubuntu/9nblggh4msv6?activetab=pivot:overviewtab)
+- Clone Erdapfel in the folder of your choice (ex: "C:\www\") using [Github Desktop](https://desktop.github.com/) or the terminal: 
+```
+sudo apt install git
+git clone https://github.com/QwantResearch/erdapfel.git
+```
+
+- Open a Ubuntu bash in this folder (shift + right-click > Open Linux CLI here)
+- Install tools, libraries and Python installer:
+```
+sudo apt update && sudo apt install -y build-essential git libexpat1-dev libssl-dev zlib1g-dev \
+libncurses5-dev libbz2-dev liblzma-dev \
+libsqlite3-dev libffi-dev tcl-dev linux-headers-generic libgdbm-dev \
+libreadline-dev tk tk-dev
+curl https://pyenv.run | bash
+```
+
+- Edit ~/.bashrc (ex: ```nano ~/.bashrc```) and add this code at the end (replace {USERNAME} with your Windows session name):
+```
+export PATH="/home/{USERNAME}/.pyenv/bin:$PATH"
+eval "$(pyenv init -)"
+eval "$(pyenv virtualenv-init -)"
+```
+
+- Install Python and its libraries:
+```
+pyenv install 3.6.8
+pyenv install 3.7.2
```

Comment:
Out of curiosity, why do you need two versions of Python ?

Reply:
it may not be mandatory:
the thing is, I had all the difficulties in the world to make WSL (linux in windows) recognize the command "python". I tried many solutions online, and this one finally paid: https://gist.github.com/monkut/35c2ef098b871144b49f3f9979032cee

(for some reason they install 2 python versions, so I kept it, but we can remove it if you think it's ok)

Reply:
I think the linked solution describes how it's possible to switch between python versions. As far as I know, our install does not depend on a specific version of Python. So, either of them would probably be enough.